metaflow/plugins/argo/argo_workflows_deployer.py

```diff
@@ -12,6 +18,95 @@
 )
 
 
+def generate_fake_flow_file_contents(
```

Comment:
This can probably move up right since it is not specific to argo. @talsperre , would this be something we could use?

Reply:
it is kinda specific to argo in the sense that the `param_info` is recovered from the metadata information of the deployed flow on argo workflows..